profile/2019-03-01-hybrid.json

```diff
@@ -6,175 +6,472 @@
     "resource-manager": {
         "microsoft.authorization": {
             "2016-09-01": [
-                "locks"
+                {
+                    "resourceName": "locks",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2016-09-01/locks.json"
+                }
             ],
             "2016-12-01": [
-                "policyDefinitions",
-                "policyAssignments"
+                {
+                    "resourceName": "policyDefinitions",
```

Comment:
resourceType instead of resourceName ?

Reply:
@sijuman please finalize on this format. The sooner we agree on the format the better it is.

Reply:
I would prefer the previous format. Again this would break the Azure Gov Profile. I would agree with Bala that it should be "resourceType" but do we need to specify it every time for each type? Seems to be repetitive. @shahabhijeet

Reply:
@sijuman this profile format never worked for us. Plus this is additive changes and should not affect the model whoever is consuming.
From day one, the piece of feedback to your team was that this format DOES NOT work.
But due to urgency for your team to release, we went ahead with it and worked with the old format.
All languages in this repository will use the new format, so that is the reason the new format should be default.
if you think some other teams needs an old format, you are free to create a new directory and have some other format for them or some other repository.
But as I said, as this is a additive change, it should not affect anything.
Plus the new format makes it clear.

Reply:
@sijuman if you want resourceType, I will update it for that and rest of the changes should be sorted and a new PR can be submitted to make whatever change you would like to make.
This PR has paths and that is what is helpful for every language to generate and everything else change be chagned with a new PR.
And feel free to submit that PR with new changes.